### DISCUSSION TITLE
Error: Could not find Chrome (ver. 119.0.6045.105). This can occur if either

### DISCUSSION THREAD

**Discussion by venuscreats** [spatie/browsershot#807] [Bugs]
`Error: Could not find Chrome (ver. 119.0.6045.105). This can occur if either
 1. you did not perform an installation before running the script (e.g. `npm install`) or
 2. your cache path is incorrectly configured (which is: /home/devmarinesl/.cache/puppeteer).
For (2), check out our guide on configuring puppeteer at https://pptr.dev/guides/configuration.`

tried to fix this issue with few node.js versions(16/18/20) and pupeeter versions to my Laravel project (5.8). but issue is still available.  Also tried with downgrading Browsershot version to  3.47.

please help to resolve this. Thanks
![sdasddsdfsdsadasd](https://github.com/spatie/browsershot/assets/3040167/3ab29a8c-b81f-4199-b292-db5569e57670)

**Comment by venuscreats** [comment]
anyone to help?

**Comment by venuscreats** [comment]
Hello,

this post is helped to solve this issue.
https://laracasts.com/discuss/channels/general-discussion/browsershot-on-windows-puppeteer-cannot-find-chromium?page=1&replyId=919138

**Comment by bitsmyth** [comment]
@venuscreats 

I was in the same boat, and it took me an uncomfortable amount of time to figure it out. maybe somebody can save time with this approach. I created the following file within the root dir of the project:

**.puppeteerrc.cjs**
```
const { join } = require('path');

/**
 * @type {import("puppeteer").Configuration}
 */
module.exports = {
    cacheDirectory: join(__dirname, '.cache', 'puppeteer'),
};
```

and added the following script to 

**package.json**

```
    "scripts": {
        "postinstall": "npx puppeteer browsers install --install-deps"
    },
```

[reference](https://stackoverflow.com/questions/77779601/puppeteer-could-not-find-chrome-ver-119-0-6045-105-error/77784336#77784336)
